pkgs/by-name/ic/icloudpd/package.nix

```diff
@@ -72,8 +72,11 @@ python3Packages.buildPythonApplication rec {
 
   preBuild = ''
     substituteInPlace pyproject.toml \
-      --replace "setuptools==69.0.2" "setuptools" \
-      --replace "wheel==0.42.0" "wheel"
+      --replace-fail "setuptools==69.0.2" "setuptools" \
```

Comment:
Why not using pythonRelaxDeps ?

Reply:
afaik it doesn't work well with the build system components, or at least in the other python packages I maintain, I've never made it work with `setuptools`.